### ISSUE TITLE
On Zimbabwe Idai recovery site, we should load population demographics 

### ISSUE THREAD

**Issue by akahure** [onaio/gisida#411]
We should load the High Resolution Population Density Maps + Demographic Estimates from https://data.humdata.org/dataset/highresolutionpopulationdensitymaps-zwe to zimbabwe idai site here http://idairecovery-zimbabwe.onalabs.org/

**Comment by cKellyDesign** [comment]
From a Mapbox perspective this data source is a Raster layer type because the data is a high-resolution geotiff. If we can convert that to a tileset then we just need to add a new Layer Type (`raster`, not `heatmap`) to Gisida.

**Comment by cKellyDesign** [comment]
@akahure An update on the FB Population data raster layer:

### TLDR
We're making concurrent progress on the Mapbox tileset and the gisida code changes to support raster layers. 

### What we're trying to do
We are still trying to get the FB Population `geotiff` to upload to Mapbox correctly so we can render it on the map. We have to do some conversions to get the file to be compatible with Mapbox and it's proving difficult to achieve. Slowing down the process is the fact that these are very large files and we are only getting errors after we have uploaded new tilesets.

To format the `geotiff` we are applying color conversions and mercator projection translations, and seem to have only done so in a way that is compliant with Mapbox uploads with a clipped version of the original data.

@KipSigei has added some core code to support raster tilesets, we know this code is working because it was able to render this clipped version of the geotiff successfully.

### What we need
Aside from a `geotiff` that works, we also need to know what color to use for the layer. I suggest `green` so that additional `blue` (flooded damage) layers and `red` (collapsed damage) layers can blend where they overlap (`purple` = flooded + collapsed; `yellow` = dense population + collapsed; `teal` = dense population + flooded; `blackish brown` = dense population + collapsed + flooded;)

**Comment by mberg** [comment]
I've already created a vector for this.  It will be easy to stylize.

Here you go

https://studio.mapbox.com/tilesets/ona.97ca3h1w/

**Comment by mberg** [comment]
note it's only visizble from zoom level 9 and down.  if you want you can
use tippecanoe to change the shapefile to make it a higher zoom. i'll
upload the shape to gisida-views

**Comment by cKellyDesign** [comment]
@akahure is this still a requirement? If so, what's the priority?
